parm/config/gefs/config.ufs

```diff
@@ -325,8 +325,8 @@ if [[ "${skip_mom6}" == "false" ]]; then
       MOM6_RIVER_RUNOFF='False'
       eps_imesh="2.5e-1"
       TOPOEDITS="ufs.topo_edits_011818.nc"
-      if [[ "${DO_JEDIOCNVAR:-NO}" = "YES" ]]; then
-        MOM6_DIAG_COORD_DEF_Z_FILE="oceanda_zgrid_75L.nc"
+      if [[ ${RUN} == "enkfgfs" || "${RUN}" == "gdas" ]]; then
```

Comment:
I guess that works, but wouldn't that cover it:
```bash
if [[ "${RUN}" != "gfs" ]]; then
```
?

Reply:
@NeilBarton-NOAA Would you be able to gauge the the impact for `RUN=gefs`?

Reply:
@guillaumevernieres I think we want this for both GFS and GEFS so that's why I didn't use your suggestion.

Reply:
Although this is the "gfs" config, I don't think we want gfs and gefs to be that out of sync.

Reply:
@JessicaMeixner-NOAA The comment is for this file `parm/config/gefs/config.ufs`
I agree, ideally, we would want to keep them in sync.
Am I misunderstanding?

Reply:
Maybe I'm missing something? So this is GEFS, so to my knowledge you come into this routine and you would be RUN="GEFS" which means you want the "else" in this loop.  If the if statement was if ~=gfs, then you would get the 75 layers not the 40 layers which is what I assume GEFS wants to my knowledge.

Reply:
I'm adding @jiandewang I don't know the impact as I don't analyze the ocean output. I would default to keeping it to interpolate_zgrid_40L.nc for gefs

Reply:
You are right @JessicaMeixner-NOAA

Reply:
I'm late to the party here, but want to confirm we want to run this for `gdas` and `enkfgfs` but not `enkfgdas`.

Reply:
The "else" should be gfs/gefs...   so I think this is my mistake.  Maybe I should switch the logic.  @guillaumevernieres can help confirm too.